### PR TITLE
Update `web_benchmarks` package to properly support wasm.

### PR DESCRIPTION
**Pull request by eyebrowsoffire**
This fixes the web_benchmarks package so it can compile and run apps with wasm. It also updates our CI steps to use a newer version of Chrome so that WasmGC works properly.

### CODE REVIEW

**Review by yjbanov — APPROVED** [review]
lgtm

File: packages/web_benchmarks/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 1.3.0
 
 * Updates minimum supported SDK version to Flutter 3.16/Dart 3.2.
+* Support for running benchmarks with the wasm compilation target.
```
Comment (stuartmorgan) [inline packages/web_benchmarks/CHANGELOG.md:4]: "Adds support for [...]", per the repo CHANGELOG style guide.

File: packages/web_benchmarks/lib/src/compilation_options.dart [hunk 2]
```diff
@@ -7,11 +7,15 @@
 /// This object holds metadata that is used to determine how the benchmark app
 /// should be built.
 class CompilationOptions {
-  /// Creates a [CompilationOptions] object.
-  const CompilationOptions({
```
Comment (stuartmorgan) [inline packages/web_benchmarks/lib/src/compilation_options.dart:11]: Removing a constructor from a public class is a breaking change, and all of our packages follow semver. If you want to remove this, the package needs a major version bump.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/web_benchmarks/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 2.0.0
 
 * Updates minimum supported SDK version to Flutter 3.16/Dart 3.2.
+* Adds support for running benchmarks with the wasm compilation target.
```
Comment (stuartmorgan) [inline packages/web_benchmarks/CHANGELOG.md:5]: This needs a description of the breaking changes; see https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#changelog-style

**Review by stuartmorgan — APPROVED** [review]

**Pull request merged**